uat/custom-components/src/test/java/com/aws/greengrass/LogGeneratorTest.java

```diff
@@ -17,19 +16,18 @@ public class LogGeneratorTest {
     @TempDir
     static Path tempPath;
     String logFileName = "localtest";
-    String logFileExtention = "log";
-    String logWriteFreqSeconds = "0.1";
+    String logWriteFreqMs = "100";
```

Comment:
Is someone going to think this is milliseconds and not messages per second?

Reply:
It is milliseconds. 0.1 seconds is 100 milliseconds.